curriculum/challenges/english/blocks/lecture-introduction-to-javascript/672d496eca926b5df8176a67.md

```diff
@@ -5,55 +5,105 @@ challengeType: 19
+
+Data types help the program understand the kind of data it's working with, whether it's a number, text, or something else.
 
 JavaScript has several basic data types that you'll use in your programs. We'll explore each data type in greater detail in future lessons. For now, here is a brief introduction of the different data types in JavaScript.
 
 The first data type we will look at is the `Number` type.
 
-A `Number` represents both integers and floating-point values. Examples of integers include `7`, `19`, and `90`. Examples of floating point numbers include `3.14` and `5.2`. A floating point number is a number with a decimal point.
+A `Number` represents both integers and floating-point values. Examples of integers include `7`, `19`, and `90`. 
+
+**NOTE**: `console.log()` is a function that outputs information to the console, which is a part of your web browser used for debugging code. You will learn more about `console.log()` in future lessons. Also, the `//` symbols are used to add comments in your code. Comments are notes for yourself or other programmers that are ignored when the code runs.
+
+Enable the interactive editor and try changing some of the integers to see it update in the console.
+
+:::interactive_editor
+
+```js
+// Examples of integers
+console.log(3);
+console.log(5);
+console.log(-67);
+```
+
+:::
+
+A floating point number is a number with a decimal point. Examples of floating point numbers include `3.14` and `5.2`. 
+
+:::interactive_editor
+
+```js
+// Examples of floating point numbers
+console.log(3.14);
+console.log(7.2);
+console.log(-14.5);
+```
+
+:::
 
 The next data type is a `String`.
 
-A `String` is a sequence of characters, or text, enclosed in quotes. Here are two examples:
+A `String` is a sequence of characters, or text, enclosed in quotes. Here is an example string using double quotes: 
 
-```md
-"Hello, world"
+:::interactive_editor
+
+```js
+console.log("I love to code!");
 ```
 
-```md
-'JavaScript'
+:::
+
+Here is an example using single quotes:
+
+:::interactive_editor
+
+```js
+console.log('I love to code!');
 ```
 
+:::
+
+Oftentimes you will use strings to represent names, labels, alert messages, etc.
+
 Another data type used in JavaScript is the `Boolean` type.
 
-A `Boolean` represents one of two possible values: `true` or `false`. You can use a boolean to check if a page is loading, or if a user is logged in or not.
+A `Boolean` represents one of two values: true or false. For example, a program might check whether a user is logged in (true) or not (false) and change the page based on that. If the user is logged in, you probably want to show them the dashboard page. Otherwise, you will want to show them the login page.
```

Comment:
```suggestion
A `Boolean` represents one of two values: `true` or `false`. For example, a program might check whether a user is logged in (`true`) or not (`false`) and change the page based on that. If the user is logged in, you probably want to show them the dashboard page. Otherwise, you will want to show them the login page.
```

Those inside the parentheses can go either way